doc/README.md

```diff
@@ -283,35 +289,35 @@ For example:
 ```markdown
 # pkgs.coolFunction {#pkgs.coolFunction}
 
-`pkgs.coolFunction` *`name`* *`config`*
+`pkgs.coolFunction` _`name`_ _`config`_
 
 Description of what `callPackage` does.
 
-
 ## Inputs {#pkgs-coolFunction-inputs}
 
 If something's special about `coolFunction`'s general argument handling, you can say so here.
 Otherwise, just describe the single argument or start the arguments' definition list without introduction.
 
-*`name`* (String)
+_`name`_ (String)
 
 : The name of the resulting image.
 
-*`config`* (Attribute set)
+_`config`_ (Attribute set)
 
 : Introduce the parameter. Maybe you have a test to make sure `{ }` is a sensible default; then you can say: these attributes are optional; `{ }` is a valid argument.
 
-  `outputHash` (String; _optional_)
```

Comment:
This is wrong. It's supposed to be a term-definition in the paragraph that is part of the outer definition.

Reply:
I don't think nested term-definition are officially supported in markdown. #investigating

Reply:
I added a `-` (list item), would it be acceptable for you?

Reply:
I guess I can mark this as solved as I updated the current documentation before doing the formatting so that this issue won't happen again. Hope it's OK now.

Reply:
Hm that seems to be a deficiency of any given markdown implementation (there's no such thing as "official" markdown). The semantics is valid DOM, so in a sense "not our problem". Making it a list item defies the original purpose.

It also seems inappropriate to iron over the work of @danielsidhion and others who have put effort into adding more structure, just because some tool happens not to know that structure. This may appear like a minor thing, but I think it's sort of at the heart of what we're doing with Nixpkgs: we're building a knowledge base, and that's a big data structure. Removing information because some tools can't handle it seems wrong.

Is it possible to disable that particular behavior? Since the other changes look alright (haven't checked everything).

Reply:
Right I hear you and fully agree. I'll write a post describing precisely what has been done so far to fix this issue at the end of the day, with screenshots and detailed explanation so we can decide what to do about this.

Reply:
Thanks a lot @drupol!

Reply:
I updated the OP with full details, explanation and screenshots. Let me know what you think.